src/PokeList.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
+
+namespace Pokedex;
+
+use Pokedex\DbConnection;
+
+class PokeList {
+
+    private $list;
+
+    /**
+     * Creates a new pokeList using a DBConnection
+     * @param \PDO $connection The particular database object to use for retrieving the list of pokemon.
+     */
+    public function __construct(\PDO $connection)
```

Comment:
Consider renaming $connecting to $db to match index.php DbConnection.php

Reply:
changed in 1b69d10